i18n-sample/gulpfile.js

```diff
@@ -0,0 +1,116 @@
+// This is based off https://github.com/Microsoft/vscode/blob/master/build/gulpfile.extensions.js
+// but simplified for the single extension use-case
+
+"use strict";
+const es = require('event-stream');
+const filter = require('gulp-filter');
+const fs = require('fs');
+const gulp = require('gulp');
+const glob = require('glob');
+const nlsDev = require('vscode-nls-dev');
+const path = require('path');
+const rimraf = require('rimraf');
+const shell = require('shelljs');
+const tsb = require('gulp-tsb');
+
+const srcBase = path.join(__dirname, 'src');
+const src = path.join(srcBase, '**');
+const typeDefinitions = path.join(__dirname, 'node_modules/**/*.d.ts');
+const out = path.join('./', 'out');
+const i18n = path.join('./', 'i18n');
+const allErrors = [];
+let startTimer = null;
+let count = 0;
+
+// TASK: Add your supported languages here
+const languages = ['jpn'];
+
+function stripSourceMappingURL() {
+  const input = es.through();
+
+  const output = input.pipe(
+    es.mapSync(f => {
+      const contents = f.contents.toString('utf8');
+      f.contents = new Buffer(
+        contents.replace(/\n\/\/# sourceMappingURL=(.*)$/gm, ''),
+        'utf8'
+      );
+      return f;
+    })
+  );
+
+  return es.duplex(input, output);
+}
+
+function createPipeline(build) {
+  const tsOptions = require(path.join(__dirname, 'tsconfig.json'))
+    .compilerOptions;
+  tsOptions.verbose = false;
+  tsOptions.sourceMap = true;
+
+  tsOptions.inlineSources = !!build;
+  const compilation = tsb.create(tsOptions);
+
+  return function() {
+    const input = es.through();
+    const tsFilter = filter(['**/*.ts', '!**/vscode-nls-dev/lib/**'], {
+      restore: true
+    });
+
+    const output = input
+      .pipe(tsFilter)
+      .pipe(compilation())
+      .pipe(build ? nlsDev.rewriteLocalizeCalls() : es.through())
```

Comment:
Is using`vscode-nls-dev` always going to require the use of gulp in the build pipeline?